src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.ts

```diff
@@ -85,6 +87,36 @@ export default class PaypalCommercePaymentProcessor {
         return this._paypalButtons;
     }
 
+    renderFields(apmFieldsContainer: string, fundingKey: keyof PaypalCommerceSDKFunding, apmFieldsStyles?: PaypalFieldsStyleOptions, fullName?: string, email?: string): PaypalCommerceFields {
```

Comment:
this method seems to have a bit too many fields. I would recommend creating an interface
`RenderApmFieldsParams` and pass a single object rather than a lot of arguments.

you could then do something like

```js
renderFields({
  apmFieldsContainer,
  fundingKey,
  apmFieldsStyles,
  ...
}: RenderApmFieldsParams): PaypalCommerceFields